src/libs/ReportUtils.js

```diff
@@ -855,6 +855,29 @@ function getDisplayNamesWithTooltips(participants, isMultipleParticipantReport)
     });
 }
 
+/**
+ * @param {Object} report
+ * @param {String} report.iouReportID
+ * @param {Object} moneyRequestReports
+ * @returns {Number}
+ */
+function getMoneyRequestTotal(report, moneyRequestReports = {}) {
+    if (report.hasOutstandingIOU) {
+        const moneyRequestReport = moneyRequestReports[`${ONYXKEYS.COLLECTION.REPORT}${report.iouReportID}`];
+        if (moneyRequestReport) {
+            return moneyRequestReport.total;
+        }
```

Comment:
Is this right? Don't we still want to run the logic below for moneyRequestReports?

Reply:
Ahh yes. You're right!